cmd/upgrade/upgrade.go

```
@@ -0,0 +1,355 @@
+		return nil, err
+	}
+	defer r.Close()
+
+	for _, f := range r.File {
+		name := filepath.Base(f.Name)
+		if name == "sec" || name == "sec.exe" {
+			rc, err := f.Open()
+			if err != nil {
+				return nil, err
+			}
+			defer rc.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `rc.Close` is not checked

(errcheck)